site/en/docs/devtools/settings/preferences/index.md

```diff
@@ -0,0 +1,686 @@
+---
```

Comment:
Somewhere in the Settings docs (this or others), we should mention that these settings are accessible through Command Menu. 

Some settings like "show ruler on hover" are more common to toggle with the shortcut

Reply:
Mentioned it: https://github.com/GoogleChrome/developer.chrome.com/pull/5274/commits/8fd4f15dc05f4bcb379eaa43d5762da6f1f972d7